widgetastic_manageiq/__init__.py

```diff
@@ -5374,5 +5374,61 @@ def select(self, name):
         return False
 
 
+class RolesSelector(Widget):
+    """This widget for custom button role selection"""
+
+    ROOT = "//div[@class='col-md-10']//table"
+    CELLS = "./tbody/tr/td"
+    INPUTS = "./tbody/tr/td/input"
+
+    def __init__(self, parent, logger=None):
+        Widget.__init__(self, parent=parent, logger=logger)
+
+    @property
+    def _elements(self):
+        br = self.browser
+        return {
+            br.text(name): el for name, el in zip(br.elements(self.CELLS), br.elements(self.INPUTS))
+        }
+
+    @property
+    def roles(self):
+        """all available roles"""
+        return list(self._elements.keys())
```

Comment:
unnecessary `list()` call here, `keys()` will return a list even on an empty dict.

Reply:
for `py2` it will return `list` but for `py3` `dict_keys` so for compatibility, we need to convert.